.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "lib/openzeppelin-contracts"]
 	path = lib/openzeppelin-contracts
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
```

Comment:
Is openzeppelin upgrade necessary? Why to `master` and not to one of official releases? I'm fine with the upgrade, but maybe it'd make sense to make it a separate PR not to mix potential issues and to make it perfectly clear what's happening.